docs/clients/brushless_drive.rst

```diff
@@ -105,72 +105,116 @@ Message Table
+| 45     | volts_at_max_amps             | get            | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 46     | slew_volts_per_second         | get, set, save | float     | V        |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 47     | slew_enable                   | get, set, save | float     | V        |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 48     | motoring_supply_current_limit | get, set, save | float     | fix16_t  |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 49     | regen_supply_current_limit    | get, set, save | float     | fix16_t  |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 50     | supply_current_limit_enable   | get, set, save | uint8     | bool     |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 51     | regen_limiting                | get            | uint8     | bool     |                                                                                                                                      |      |
```

Comment:
Description: "When dynamic supply current limiting is enabled, this reports whether the drive voltage is currently being limited due to the regeneration current back through the supply. True means that the limiter is currently controlling the drive voltage because the regeneration supply current is at or above the limit, false means the regeneration current is low enough that the limiter has not activated".

No units besides boolean